flytekit/remote/workflow.py

```diff
@@ -1,16 +1,17 @@
+from __future__ import annotations
```

Comment:
we don't need this, right?

Reply:
nope, but i like it.  so that i don't have to do -> "MyClass"

Reply:
https://www.python.org/dev/peps/pep-0563 search for "Annotations using nested classes and their respective state are still valid. They"